src/khoj/interface/web/content_type_input.html

```diff
@@ -1,75 +1,82 @@
-{% extends "base_data_integration.html" %}
+{% extends "base_config.html" %}
 {% block content %}
-<h2>{{ content_type }}</h2>
-<form id="config-form">
-    <div id="success" style="display: none;" ></div>
-    <table>
-        <tr>
-            <td>
-                <label for="input-files">Input Files</label>
-            </td>
-            <td id="input-files-cell">
-                {% if current_config['input_files'] is none %}
-                    <input type="text" id="input-files" name="input-files">
-                {% else %}
-                    {% for input_file in current_config['input_files'] %}
-                        <input type="text" id="input-files" name="input-files" value="{{ input_file }}">
-                    {% endfor %}
-                {% endif %}
-            </td>
-            <td>
-                <button type="button" id="input-files-button">Add</button>
-            </td>
-        </tr>
-        <tr>
-            <td>
-                <label for="input-filter">Input Filter</label>
-            </td>
-            <td id="input-filter-cell">
-                {% if current_config['input_filter'] is none %}
-                    <input type="text" id="input-filter" name="input-filter">
-                {% else %}
-                    {% for input_filter in current_config['input_filter'] %}
-                        <input type="text" id="input-filter" name="input-filter" value="{{ input_filter }}">
-                    {% endfor %}
-                {% endif %}
-            </td>
-            <td>
-                <button type="button" id="input-filter-button">Add</button>
-            </td>
-        </tr>
-    </table>
+<div class="page">
+    <div class="section">
+        <h2 class="section-title">
+            <img class="card-icon" src="/static/assets/icons/{{ content_type }}.svg" alt="{{ content_type|capitalize }}">
+            <span class="card-title-text">{{ content_type|capitalize }}</span>
```

Comment:
What is the `|capitalize` for? I'm assuming that capitalizes the text in jinja-langauge?